MATLAB/rl_ofdm_calib.m

```diff
@@ -33,6 +33,10 @@
 bs_ids = string.empty();
 bs_sched = string.empty();
 
+if TX_GN > 81
```

Comment:
Name 81.     Maybe place the definition in a common file so you don't have multiple definitions in different locations
Use parenthesis because I think they are awesome.